selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/refactor/move/MoveItemsTest.java

```diff
@@ -106,7 +118,10 @@ public void checkMoveItem0() throws Exception {
     refactor.clickOnExpandIconTree("/src/main/java");
     refactor.chooseDestinationForItem("p1");
     refactor.clickOkButtonRefactorForm();
+    loader.waitOnClosed();
     refactor.waitMoveItemFormIsClosed();
+    loader.waitOnClosed();
+    events.waitExpectedMessage(APPLY_WORKSPACE_CHANGES, LOAD_PAGE_TIMEOUT_SEC);
```

Comment:
Is load page timeout the right number here? What would be a reasonable timeout for those refactorings?

Reply:
I've changed to `DEFAULT_TIMEOUT`, the value is the same - 10 sec. It should be enough to apply changes.